src/scripts/make-configs.sh

```diff
@@ -16,6 +16,9 @@ Secrets Configuration
  This bash file is sourced and expected to set the following variables
  - KEYSTORE_PASSWORD
  - PPMS_URL
+ - VLER_KEY_PUBLIC
+ - VLER_KEY_PRIVATE
+ - VLER_URL
```

Comment:
Add the two truststore variables to this script.

Reply:
Also placed under this comment spot